redditdownload/tests/test_redditdownload.py

```diff
@@ -3,8 +3,10 @@
 
 import unittest
 from unittest import TestCase
+import os, sys
 from os import getcwd
-
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
does it have to append it to system path? if you use nose or pytest i think it can find it.